src/Support/SupportData.php

```diff
@@ -257,29 +213,42 @@ public function get_site_info() {
 
 		global $wpdb;
 
-		$wp_type = is_multisite() ? ( defined( 'SUBDOMAIN_INSTALL' ) && SUBDOMAIN_INSTALL ) ? 'subdomain' : 'subdir' : 'single';
+		$wp_type = 'single';
+
+		if ( is_multisite() && ( defined( 'SUBDOMAIN_INSTALL' ) && SUBDOMAIN_INSTALL ) ) {
+			$wp_type = 'subdomain';
+		} elseif ( is_multisite() ) {
+			$wp_type = 'subdir';
+		}
 
 		$active_theme = wp_get_theme();
 		$active_theme = static::normalize_theme_info( $active_theme );
 
-		$amp_settings = \AMP_Options_Manager::get_options();
+		$amp_settings = AMP_Options_Manager::get_options();
 		$amp_settings = ( ! empty( $amp_settings ) && is_array( $amp_settings ) ) ? $amp_settings : [];
 
 		$loopback_status = '';
 
-		if ( class_exists( 'Health_Check_Loopback' ) ) {
-			$loopback_status = \Health_Check_Loopback::can_perform_loopback();
+		if ( class_exists( 'WP_Site_Health' ) ) {
+			$site_health     = new WP_Site_Health();
```

Comment:
Would be best to get the `WP_Site_Health` instance if possible.

```suggestion
			if ( method_exists( 'WP_Site_Health', 'get_instance' ) ) {
				$site_health = WP_Site_Health::get_instance();
			} else {
				$site_health = new WP_Site_Health();
			}
```